packages/common/client-utils/src/storageUtils.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IPartialSnapshotWithContents } from "@fluidframework/driver-definitions";
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+/**
+ * Utility API to check if the type of snapshot contents is `IPartialSnapshotWithContents`.
+ * @alpha
+ * @param obj - obj whose type needs to be identified.
+ */
+export function instanceOfIPartialSnapshotWithContents(
```

Comment:
Gut feeling would make me push this towards driver-utils instead; seems more in line with my understanding of the layering, and it already has the dependencies we're adding to client-utils. Is there a reason why the new functions can't go there?

Reply:
Hmm, just checked. I thought runtime does not depend on driver utils as it in loader. So, added here. But it does depend on driver utils. We could move it there.